lib/workers/repository/updates/flatten.ts

```diff
@@ -216,8 +217,14 @@ export async function flattenUpdates(
       update.semanticCommits = semanticCommits;
     }
   }
-  return updates
-    .filter((update) => update.enabled)
+  const filteredUpdates = updates
+    .filter((update) => update.enabled !== false)
     .map(({ vulnerabilityAlerts, ...update }) => update)
     .map((update) => filterConfig(update, 'branch'));
+  if (filteredUpdates.length < updates.length) {
+    logger.debug(
+      `Filtered out ${updates.length - filteredUpdates.length} disabled update(s). ${filteredUpdates.length} update(s) remaining.`,
```

Comment:
```suggestion
      `Filtered out disabled ${updates.length - filteredUpdates.length} disabled update(s). ${filteredUpdates.length} update(s) remaining.`,
```
IMO this makes it clearer for users why they are filtered out.

Reply:
duplicate disabled ☝️